omero/sysadmins/public.rst

```diff
@@ -138,3 +139,131 @@ authentication. For example a direct link to image will look as follows::
+treatment conditions represented in each figure panel.
+For example, Project
+`Schleicher_etal_figure7_c <https://omero.lifesci.dundee.ac.uk/webclient/?show=project-27920>`_
+contains images underlying the
+`publication Figure panel 7c <http://rsob.royalsocietypublishing.org/content/royopenbio/7/11/170099/F7.large.jpg>`_.
+Some Projects underlie two publication figure panels, such as Project
+`Schleicher_etal_figure2_a_c <https://omero.lifesci.dundee.ac.uk/webclient/?show=project-27917>`_
+where representative images are shown in panel a and the
+corresponding quantification is shown in panel c of `Figure 2 <http://rsob.royalsocietypublishing.org/content/royopenbio/7/11/170099/F2.large.jpg>`_
+This makes clear which original images are underlying which figure panels
+in the publication.
+
+Data can also be tagged with OMERO tags to enhance the browsing
+possibilities through these data for any user with basic knowledge of
+OMERO. For example, see `Tag:Schleicher_etal_figure1_a <https://omero.lifesci.dundee.ac.uk/webclient/?show=tag-364188>`_. The
+tags are highlighting the images displayed in the publication figures as
+images. The other, non-tagged images in the group are the ones used for
+analysis which produced the published numerical data.
+
+Key-Value pairs can be used to add more detailed information about the
+study and publication. For example, go to `Schleicher_etal_figure1_a <https://omero.lifesci.dundee.ac.uk/webclient/?show=project-27936>`_
+and expand the 'Key-Value Pairs' section in the right-hand pane to display
+the content (see the :help:`Managing data guide <managing-data.html#keyvalue>` for information on using Key-Value pairs).
+
+Configuring URLs
+^^^^^^^^^^^^^^^^
+
+The URL of the first Project (corresponding to the first
+figure in the publication) can be used for a DOI and data landing
+page. For example, Project 'Schleicher_etal_figure1_a'
+`<https://omero.lifesci.dundee.ac.uk/webclient/?show=project-27936>`_
+corresponds to `<http://dx.doi.org/10.17867/10000109>`_.
+
+Optionally, you can decide on a set pattern of URLs for this and future
+publications. For example, in Dundee we have established a pattern which
+supposes every new publication from our institution will be in a separate
+group, and this group will be directly navigable by the public user using
+the syntax: “server-address/pub/publication-identifier”. This means for
+example, `<https://omero.lifesci.dundee.ac.uk/pub/schleicher-et-al-2017>`_
+is the link where "omero.lifesci.dundee.ac.uk" is the server address, and
+"schleicher-et-al-2017" is the publication-identifier.
+
+This makes use of redirects allowing
+`<https://omero.lifesci.dundee.ac.uk/pub/schleicher-et-al-2017>`_ to
+link to the correct group and Project in OMERO, just as the
+DOI above does. Redirects need to be set in the
+`NGINX <http://nginx.org/>`_ component of the OMERO.web installation
+dedicated to publication workflows.
```

Comment:
@joshmoore  - worth linking to our example at https://github.com/openmicroscopy/prod-playbooks/blob/master/nightshade-webclients.yml#L172 ?